src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -143,14 +143,17 @@ public static MauiApp CreateMauiApp()
 				.ConfigureEssentials(essentials =>
 				{
 					essentials
-						.UseVersionTracking()
 						.UseMapServiceToken("YOUR-KEY-HERE")
 						.AddAppAction("test_action", "Test App Action")
 						.AddAppAction("second_action", "Second App Action")
 						.OnAppAction(appAction =>
 						{
 							Debug.WriteLine($"You seem to have arrived from a special place: {appAction.Title} ({appAction.Id})");
 						});
+
+					// TODO: Unpackaged apps need to know the package ID and local data locations
+					if (AppInfo.IsPackagedApp)
+						essentials.UseVersionTracking();
```

Comment:
Right now, the `VersionTracker` uses preferences which is not yet "ported" because it makes use of a few features that rely on or assume a packaged app.